src/test/java/SetTest.java

```diff
@@ -20,12 +21,17 @@ public void setUp() {
   @ParameterizedTest
   @ValueSource(ints = {1, 2, 3})
   void _1_2_3을_가진다(int value) {
-    assertThat(numbers.contains(value)).isTrue();
+    assertThat(numbers).contains(value);
   }
 
   @ParameterizedTest
   @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
   void csvsource로_구성되어있다(int value, boolean expected) {
     assertThat(numbers.contains(value)).isEqualTo(expected);
   }
+
+  @Test
+  void check_set_size(){
+    assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함